### ISSUE TITLE
[FEATURE] Grave Compass

### ISSUE THREAD

**Issue by ginsm** [ginsm/forgotten-graves#2]
**Is your feature request related to a problem? Please describe.**
An user could potentially die and have the server shutdown, client crash, etc before they get a chance to look at the death coordinates.

**Describe the solution you'd like**
A command (`/graves lookup [user]` -- user is optional) should let you view active grave locations (which would be removed from this list upon retrieval).

**Additional context**
This command could extend to regular users as a means for them to look up their own grave locations.

**Comment by ginsm** [comment]
I want to implement this still but in a different fashion -- I'd like to add a compass that you can craft that opens a GUI when you right click that lets you select a grave to point to within a given dimension.

**Comment by Rebel459** [comment]
Wouldn’t a recovery compass handle most of that survival-wise? I understand adding a command, but recovery compasses would work in most cases. I suppose, rather than adding a new compass, you could utilise the recovery compass and have it use the GUI to point to a specific grave.

**Comment by ginsm** [comment]
Yup, that's what I ultimately decided. I recently implemented storing the graves and the like, so that much has been handled. Now, I just need to figure out the UI/UX of the GUI and how to create it.

**Comment by ginsm** [comment]
This is just a mock up but this is what the GUIs will look like for the tracking and admin panels once implemented:

![GraveTrackingGUI](https://github.com/ginsm/forgotten-graves/assets/52685959/ca376ef0-d756-481c-a0bc-0673d7f1303c)
![GravesAdminPanelGUI](https://github.com/ginsm/forgotten-graves/assets/52685959/c438b9d4-60d7-4188-b7d2-5817ba6b7b5b)

I'll resume work on actually implementing it after #93 is finished.
